### PR TITLE
/index.html: Mention focus on stability, not bleeding-edge.

### PR DESCRIPTION
**Pull request by flexibeast**
This PR is accompanied by https://github.com/void-linux/void-docs/pull/349. Suggestions for better wording/phrasing welcome. :-)

### CODE REVIEW

**Comment by Vaelatern** [comment]
`less fear` begs comparison. The original wording for the end of that paragraph was fine.

**Comment by flexibeast** [comment]
@Vaelatern: Point taken. My concern with the existing end of that para is that i think "always be up-to-date" can be read as "when you update, your system will always have the latest release of all software", given that *all* distros are as up-to-date as they can be after updates are released and applied, and this page is discussing distinguishing features.

**Comment by flexibeast** [comment]
(i.e. Debian could also say "apply any available updates daily, and your system will always be up-to-date".)

**Comment by Vaelatern** [comment]
"Install once, update routinely and safely." ?

**Comment by ahesford** [comment]
Void focuses on stability, rather **than** being bleeding-edge.

Also, I like @Vaelatern 's suggestion for "Install once [...]".

**Comment by flexibeast** [comment]
Yeah, i think @Vaelatern's suggestion is better than what i've currently got; happy to change it to that, if others agree.

**Comment by Vaelatern** [comment]
@void-linux/webmasters  ?

**Comment by flexibeast** [comment]
@ahesford: Oh, good catch re. "than", thanks.

File: index.html [hunk 1]
```diff
@@ -37,8 +37,8 @@ <h3>Not a fork!</h3>
 			<p>Unlike trillions of other existing distros, Void is not a modification of an existing distribution.  Void's package manager and build system have been written from scratch.</p>
 		</div>
                 <div class="col-md-4">
-                        <h3>Rolling release</h3>
-			<p>Install once, update daily. Your system will always be up-to-date.</p>
+                        <h3>Stable rolling-release</h3>
```
Comment (ericonr) [inline index.html:40]: I'm not sure this has a hyphen.

Comment (flexibeast) [reply index.html:40]: Thanks, fixed.

**Comment by flexibeast** [comment]
i've just updated the PR with the corrections from @ahesford and @ericonr, and the suggestion from @Vaelatern.

**Pull request merged**